### PR TITLE
(SIMP-MAINT) Add FAQ on Puppet certificate errors

### PR DESCRIPTION
**Pull request by trevor-vaughan**

### CODE REVIEW

File: docs/help/FAQ/Puppet.rst [hunk 1]
```diff
@@ -5,6 +5,37 @@ Puppet-Related Issues
...
+#. Check that the connection from the client system to the server can
+   successfully connect:
+
+   .. code-block:: bash
+
+      openssl s_client -host $(puppet config print server) \
+      -port $(puppet config print masterport) \
+      -cert $(puppet config print hostcert) \
+      -key $(puppet config print hostprivkey) \
+      -CAfile $(puppet config print cacert)
```
Comment (lnemsick-simp) [inline docs/help/FAQ/Puppet.rst]: When I tried this on the puppet master, it worked.  When I tried it on another host it did not. openssl could not `read /etc/puppetlabs/puppet/ssl/ca/ca_crt.pem` in the `X505_load_cert_crl_file` routine.

Comment (judyj) [reply docs/help/FAQ/Puppet.rst]: worked for me!

**Comment by jeannegreulich** [comment]
Trevor, this only works for me from a client unless I specify a cipher.  (it works fine from the puppet server.)   I thought maybe it FIPS (seris FIPS, client is not FIPS) so I went to a client that also had FIPS enable and got the same problem.    I did openssl cipher on both and listed the ciphers and chose the first one that matched.  It looks like is negotiating EDHE-RSA-AES256-SHA, which is in the list of both machines but that doesn't work.   (SHA needs more definition perhaps.)

File: docs/help/FAQ/Puppet.rst (hunk 1 above)
Comment (jeannegreulich) [reply docs/help/FAQ/Puppet.rst]: Doesn't work for me.  the puppet ca cert seems to be at //etc/puppetlabs/puppet/ssl/certs/ca.pem on our systems.  Don't know if it is our setup or SIMP or puppet.  Need to look at pupmod to see what it does.

Comment (jeannegreulich) [reply docs/help/FAQ/Puppet.rst]: Found it, need to use localcacert.  I will change it.  cacert is the location on the server.

**Pull request merged**